src/gr11review/part2/Utility.java

```diff
@@ -1,5 +1,274 @@
+    * @param String 2  The last letter
+    * @param String 2  Both letters 
+    * @author Adsana 
+    */
+
+
+
+    // Create a method to print out first and last letters of a 3 letter word
+    // Use substring to print out the letters we want
+    public static String zipZap(String strWord){
+      String strLetter1;
+      String strLetter2;
+      String strTwoLetters;
+     
+      strWord = "";
+      strLetter1 = strWord.substring(0);
+      strLetter2 = strWord.substring(2);
+      strTwoLetters = strLetter1 + strLetter2; 
+
+    
+    return strTwoLetters;
+    }
+
+     /**
+    * print longest word in file
+    * @param String 1  The words 
+    * @param String 2  The longest word
+    * @author Adsana 
+    */
     
+    TextInputFile words = new TextInputFile("words.txt");
+
+    public static String longestword(String strWordstxt){
+        String strWords;
+        String strLongestWord;
+        // finding the longest word in the file 
+        // no word can be longer than the longest word so will print the highest one
+        if (strWords.length() > strLongestWord.length()){
+          strWords = strLongestWord;
+        }
+    return strLongestWord;
+    }
+
+    /**
+    * continue multiples of 10 in each row
+    * @param Nums  The numbers in the array
+    * @param Count  counter variable
+    * @author Adsana 
+    */
+
+    public static int[] tenRun(int[] nums){
+      int intNums;
+      int intCount;
+
+      for(intCount = 0; intCount < intNums.length(); intCount++){
+         // Check if numbers are multiples of 10
+         intNums[intCount] % 10 == 0;
+          // check if the net number is a multiple of 10
+          if(intNums[intCount + 1] % 10 != 0){
+            intNums[intCount + 1] = intNums[intCount];
+          }
+        }
+        return intNums;
+      }
+
+      /**
+    * print true if inner numbers are in the outer numbers
+    * @param Count 1  counter variable
+    * @param Count 2  counter variable
+    * @param Inner    Inner numbers
+    * @param Outer    Outer numbers
+    * @author Adsana 
+    */
+
+    public static linearIn(int[] outer, int[] inner){
+      int intCount1;
+      int intCount2; 
+      int intInner;
+      int intOuter;
+
+      // increasing values for both outer and inner
+      for (intCount1 = 0; intCount1 < intInner; intCount1++){
+      for (intCount2 = 0; intCount2 < intOuter; intCount2++){
+          // create the restriction that both values have to be there in order for it to be true
+          if (intInner[] == intOuter[]){
+            return true;
+          }else{ 
+            return false;
+          }
+      }
+      }
+    }
+    
+    /**
+    * print Pascal triangle in grid form
+    * @param Int i  The rows
+    * @param Int j  The columns
+    * @author Adsana 
+    */
+
+    public static void pascalTri(int i, int j){
+        // Create pascal triangle in grid from using arrays 
+        int intPascalTri[][] = { {1,1,1,1,1,1,1,1,1,1},
```

Comment:
the 2D array values should be computed, not hard coded.  Method should be able to handle input dimensions of any value i.e 20x20